src/binding/c/misc_api.txt

```diff
@@ -0,0 +1,200 @@
+# vim: set ft=c:
+
+MPI_Aint_add:
+    .desc: Returns the sum of base and disp
+    .skip: ThreadSafe, Fortran, validate-ANY
+    .seealso: MPI_Aint_diff
+    .impl: direct
```

Comment:
It does not returns an error but returns `Return value: Sum of the base and disp argument`. Can we write so?

Reply:
Added `.skip: Errors` directive